public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -24,3 +37,57 @@ export async function fetchTenants(http: HttpStart): Promise<DataObject<Tenant>>
 export async function fetchTenantNameList(http: HttpStart): Promise<string[]> {
   return Object.keys(await fetchTenants(http));
 }
+
+export function transformTenantData(rawTenantData: any) {
+  const tenantList = map(rawTenantData, (v: any, k: string) => ({
+    tenant: k.startsWith('global') ? GLOBAL_USER_DICT.Label : k,
+    reserved: v.reserved,
+    description: k.startsWith('global') ? GLOBAL_USER_DICT.Description : v.description,
+    tenantValue: k.startsWith('global') ? GLOBAL_USER_DICT.Value : k,
+  }));
+  // Insert Private Tenant in List
+  tenantList.splice(1, 0, {
+    tenant: PRIVATE_USER_DICT.Label,
+    reserved: true,
+    description: PRIVATE_USER_DICT.Description,
+    tenantValue: PRIVATE_USER_DICT.Value,
+  });
```

Comment:
This part is conditional, need to read the configuration to check if private tenant is enabled

Reply:
fixed.